com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfGeometryCollision.cs

```diff
@@ -167,21 +167,27 @@ private static void ConvertMeshToColliders(GameObject gameObject, string locatio
                     templateFileName = Path.GetFileNameWithoutExtension(meshFilePath);
                     filePath = Path.GetDirectoryName(meshFilePath);
                 }
+                else if (cylinder) 
+                {
+                    filePath = "Assets/URDF/GeneratedMeshes";
+                    templateFileName = "Cylinder";
+                    RuntimeURDF.AssetDatabase_CreateFolder("Assets", "URDF");
+                    RuntimeURDF.AssetDatabase_CreateFolder("Assets/URDF", "GeneratedMeshes");
+                }
```

Comment:
1.  create the newly generated meshes under the same folder as the original URDF rather than a new folder. (similar to `GameObject CreateMeshCollider(Link.Geometry.Mesh mesh)`)

Reply:
2. create prefab only if `(!RuntimeURDF.IsRuntimeMode())`

Reply:
> create the newly generated meshes under the same folder as the original URDF rather than a new folder. (similar to GameObject CreateMeshCollider(Link.Geometry.Mesh mesh))

I was having an issue with this, as cylinder meshes in the URDF file do not have a `Geometry.Mesh` filepath or name:
```xml
<geometry>
    <cylinder length="0.018" radius="0.033"/>
</geometry>
```

whereas meshes from obj or stl do:

```xml
<geometry>
    <mesh filename="package://turtlebot3_description/meshes/wheels/right_tire.stl" scale="0.001 0.001 0.001"/>
</geometry>
```

so I wasn't able to parse a directory without doing a more involved saving of temporary paths (in the case a mesh has been generated from an stl or obj path previously), assumption of mesh generating order (an stl or obj mesh would have to go first in order to save a temporary path), or a longer chain of passing the filepath down from the moment the user clicks `Import URDF`--but also I wasn't sure if I was missing something obvious here, let me know what you think!

> create prefab only if (!RuntimeURDF.IsRuntimeMode())

Only the directories are being created in the `if cylinder` case, and the prefab is still only being called after line 188 inside the `if (!RuntimeURDF.IsRuntimeMode())` check

Reply:
Since in runtime mode we won't be creating any prefabs, we also don't need to create directories. Note that in runtime mode, the URDF may be outside the Asset directory.

Reply:
I see your point regarding the mesh filename. What are your thoughts about adding a new variable in RuntimeURDF class that is set with the current URDF path at the start? I believe it should be fine since the current code requires importing one URDF at the time and doesn't support multiple concurrent URDF loading.

Reply:
i found that the UrdfAssetPathHandler supports a static `GetPackageRoot()` function, so this is now used to create a new `meshes/` directory for cylinder meshes. do you think this is satisfactory? it will not always generate in the same location as the `stl` and `obj` meshes (which may live specifically in a `_description/` directory, or a `meshes` directory, it seems to vary per package)--but it will generate inside, for example:

```
Assets
|-- turtlebot3
    |-- Materials/
    |-- model.urdf
    |-- meshes/                  <-- here!
         |-- Cylinder.asset
         |-- Cylinder 1.asset
         |-- ...
    |-- turtlebot3_description/
        |-- meshes/
...
```

what do you think of this?

Reply:
I like this.